great_expectations/datasource/fluent/databricks_sql_datasource.py

```diff
@@ -80,14 +82,22 @@ class DatabricksSQLDatasource(SQLDatasource):
     Args:
         name: The name of this DatabricksSQL datasource.
         connection_string: The SQLAlchemy connection string used to connect to the postgres database.
-            For example: "databricks+connector://token:<token>@<host>:<port>/<database>?http_path=<http_path>"
+            For example: "databricks://token:<token>@<host>:<port>/<database>?http_path=<http_path>"
         assets: An optional dictionary whose keys are TableAsset or QueryAsset names and whose values
             are TableAsset or QueryAsset objects.
     """
 
     type: Literal["databricks_sql"] = "databricks_sql"  # type: ignore[assignment]
     connection_string: Union[ConfigStr, DatabricksDsn]
 
+    def test_connection(self, test_assets: bool = True) -> None:
+        try:
+            super().test_connection(test_assets)
+        except TestConnectionError as e:
+            raise TestConnectionError(
+                f"Could not connect to Databricks - please ensure you've installed necessary dependencies with `pip install great_expectations[databricks]`.\n\n{str(e)}"
+            ) from e
```

Comment:
I think we should check the exception here to see if it's a missing SQL Dialect error and only then re-raise this specific error.

Reply:
yup that makes sense to me - going to dig in to see how we can find the nested error